server.elm

```diff
@@ -2974,6 +3055,20 @@ lastEditScript = """
             }
           }
         );
+        if (! @(browserSide)) {
```

Comment:
`@(browserSide)` will output `True` because Elm has a capital in front of booleans. Please use something like `if browserSide then "true" else "false"` to convert to effective javascript.
Or even better, do this only once in the luca script, store the result in a javascript variable browserSide, and refer to this javascript variable instead.